backend/app/requests/adequacy.py

```diff
@@ -76,6 +76,10 @@ def adequacy_request(app, flask_request, engine):
     measurer_methods = config.get('measurer').keys()
     dataset_hint = request.get('dataset_hint', None)
```

Comment:
would `request.get('blah', None) or None` be a bit clearer? with a comment about the empty string?

Why do we have an empty string there in the first place?

Reply:
Good point -- should we just pass null values from the frontend?